app/models/booth.rb

```diff
@@ -35,13 +35,17 @@ class Booth < ActiveRecord::Base
     state :to_reject
     state :rejected
     state :canceled
+    state :confirmed
 
     event :restart do
-      transitions to: :new, from: [:withdrawn, :to_accept, :to_reject, :canceled]
+      transitions to: :new, from: [:withdrawn, :rejected, :canceled, :confirmed]
     end
     event :withdraw do
       transitions to: :withdrawn, from: [:new, :to_accept, :accepted, :to_reject, :rejected]
```

Comment:
I should be able to withdraw my booth, after I have confirmed. Why aren't you following the same transitions we have for proposals?